### DISCUSSION TITLE
A Proposal for FIL+

### DISCUSSION THREAD

**Discussion by Crypochen** [filecoin-project/notary-governance#822] [General]
FIL+ are facing the most turbulent and controversial moment, and the damage is done already. Very less LDN can be signed during these weeks , the total datacap sealed everyday dropped from 3.6P to 1.8P, initial pledge increased quickly than ever this month. **We do need to solve this problem soon!**

There are 3 things for sure in the first place.
1. Any unrelated LDNs that have CID sharing issue should be canceled. Unrelated LDN that shared the same CID is self- sealing and cheating undisputed. In the case of CID sharing, client is not real, data is faked, data is not transferred, copies are not spread or spread in a limited circle.
2. No abuse and blame to person/corporation, Dcent team have did everything they can to avoid fraud in FIL+ project. And the CID sharing LDNs are surely fraud. It has the effect of bad money driving out good money: cheating SPs has the lowest cost. This will drive away honest Sp and harm the communty.
3. We need to focus on what we should do in the future. Always keep building is a better way for FIlecoin Network now.

We need to discuss what FIL+ should be in the future, and how it will help the whole ecology.
Firstly, we will discuss the paid deals and the unpaid deals controversy.
Secondly, we will discuss the Large Sps and Datacap supply.
Thirdly, we will discuss the 4 types of  FIL+ and why there is so much fraud in LDN Project.
Fourthly, we will discuss the priority of datacap usage and see what we can do to move forward again.

It is only a discussion now, but i do think we need to face the difficult situation now.

**Paid deals and Unpaid deals**
I have seen some community members who have contributed a lot to filecoin say that we should cancel FIL+, because these deals are not paid, or because of the huge damage it caused.
No one will deny that the filecoin business model depended on paid deals in the long future. But i also have to admit that filecoin needs time to build and find its own way to grow.
Without the FIL+ Project, I do not think that most SPs could survive this bear market, especially the big ones, and I do not think the price would be satisfactory for most of us. The filecoin ecology needs these infrastructures(storage, sealing machine, experienced developer) to keep on running, needs the tokens to be staked onchain.
We do need to look for ways to get more paid deals and we do need to keep on sealing the unpaid deals.

**Large SPs and Datacap**
Large SPs are even harder in the bear market. Their sealing power may be higher then a hundred TiB or even thousand TiB per day, and they may pay 100K or even a million USD for the data center every month.
1. Their fixed cost needed to be paid monthly is super high and sealing CC sector is Loss-making.
2. They may have or buy millions token waiting for seal. Waiting for unstable supply of datacap to seal is horrible. They want to seal quickly and stable.
3. If they leave the ecology, the network QAP will be lower and the price will be even lower than now. Most of the large Sps are corporations that attract investors to invest in filecoin, not other coins or stock.
6. Most of them are not good at onboarding client who are willing to pay for storage. So they are not good at looking for paid deal. Their role should be building the infrastructure and there should be roles like DP to onboard paid clients and send these deals to Sps. Even buying datacap is reasonable for sps’ business model.
7. They are willing to pay for Datacap. This can also be a way to subsidize clients.

From the perspective of the filecoin community, shall we keep large SPs alive？
If we want to keep them alive, we need to find a way to supply stable datacap to these corporations. And they just keep on helping the ecolody by building infrastructure, sealing and staking tokens. If not , No matter how strict your censorship is, they are both notaries and SPs, they will still find ways to cheat.
If not , we cancel the FIL+ project and keep on finding a way to paid deals business model.

**The four types of FIL+ Project**
There’s 4 types of FIL+ projects which have the ability to allocate datacap to SPs.
 
_Notary model_
Notary can allocate datacap to Sps depending on their own decision. As notaries do not have large datacap to allocate , we do not discuss this model here. There may be fraud, but the total number is so small.
 
_SlingshotV3 model_
SlingshotV3 deal engine will give datacap to Sps that have downloaded the car files which are made by Data Preparer.
In this model,  the data is true . Everyone who downloads the data can get datacap. It is the most decentralized model I saw in allocating datacap to SPs.
In this model, DP are most important. They need to transfer data to sps. Sps downloading from sps that have sealed datacap is low efficiency.
 
_LDN model_ 
LDN model starts when a client applies for LDN on GitHub and they will find the sps so that they can transfer their data to SPs and send deals to their node.
LDN model assumes that clients are willing to apply for datacap, and they have the ability to do the complex work to apply LDN,  make car files, upgrade their bandwidth, transfer raw data, look for notaries to sign, send deals on chain and more.
Actually, the complex process is negative incentive to clients, and the filecoin network is still building so that they can not compare filecoin services to the AWS services.
Facing the complex process of LDN and the Low quality service, though it is zero fee, i do not see a way that clients are incentive to use filecoin.
In the situation of LDN, DP and SP are two types of roles that help the client to do complex work.
Most DP deal with the open data from university, NGO, Institute.  Most dataset have been used in slingshot projects.
Most SP deal with the enterprise data from the client they onboarding.
I have to admit that DPs really download the raw data and make car files and transfer the data to SPs. If you have done this , you will understand what a lot of complex work it takes to send and seal one copy to other sps.
And I have to admit that most CID sharing is found when Sps are helping the client. They fake the client and fake the data and fake everything. As most of the big SPs are notaries, it is so simple for a gruop of notaries to cheat and make tons of money.
 
I think the unfairness is caused by some SPs faking everything and getting the datacap at super low cost, while others are costing more and mining less block reward.

_E-FIL+ model_
E-FIL+ is still building and we will talk about this in the future.
But I think they need to develop paid deals. 

**Summary** 
Despite the Notary model, if we want to rebuild FIL+ Project, we need to find out a way to consider both onboarding the paid client and the stable datacap supply to  Sps. Datacap is an incentive for all the sps, and we need to use it to secure and boost our ecology.
How can we rebuild the LDN project, slingshotV3 project and the E-FIL+ project?

**Brief solution**
- The LDN project should close at the end of 2023 
  - Compare to slingshotV3, they are all unpaid deals, but it is centralized and  causes a lot of damage to our community.
  - The unrelated LDNs have CID sharing issues and should close ASAP.
  - The rest LDNs that have no CID issue should keep on sealing to keep the stability of the network ASAP. But we need a new way to stop cheating. Hidde has done a lot of work already.（TBD）
  - If there’s no LDNs, I think the notary should focus on the E-FIL+ or we just do not need any notaries.
- The SlingshotV3 project should keep on developing and help to produce a stable datacap supply to all the sps. 
  - The dataset from slingshotV3 is valuable for all humankind, they are worth storing.
  - All the dataset from slingshotV3 should be made of car files for downloading and can be downloaded by anyone and can be sealed in any datacenter.  After one sps successfully seals one copy, they can keep on sealing the second one. 
    - The Sps have the largest bandwidth and sealing power can download first, they have the liability to let others download it from their data center.
  - All SPs should do a KYC to provide the details of the data center. They especially need to have decent bandwidth to download the whole dataset or how they will get the car files(from which datacenter offline)
  - All SPs need to save an unsealed copy and open fast retrieval.
- The E-FIl+ needs to use DataCap as a weapon to develop the paid client. 
  - Sps sealing SlingshotV3 Model should pay a tiny token（0.5fil/T for example） to a pool which will be used for developing the paid client. These tokens can be used as an investment fund for clients onboarding startups, customer incentives to use filecoin and more.
  - 20X/30X QAP can be used for sps sealing paid deals

**Comment by mjroddy** [comment]
Agree on many things here but I would suggest simplifying discussion to get traction faster.

“Filecoin Plus is a program that aims to maximize the amount of useful storage that Filecoin can and will support.

The ultimate goal of Filecoin Plus is to accelerate the timeframe over which storage goods and services on Filecoin become more useful and attractive. The program plays an essential role in shaping the product offering on the network in collaboration with storage miners.”

Having gone through many applications recently many of them do not go close to fitting this

Possibly FIL+ should be thought of as an R&D grant for people working on USEFUL datasets/use cases/research that would in turn drive value back into filecoin.

In majority of cases clients need to pay for storage. This is the reality we need to move too.

**Reply by Crypochen** [reply]
Going to simplify.

In this proposal, i am discussing too much topic in poor english, hoping i can write in chinese.

I think i need to focus on a better way to take place of this LDN project: a no paid deal, fraud, controled by large entities, unfair to small sps project.
